src/Components/Publishing/Layouts/__tests__/StandardLayoutAdsEnabled.test.tsx

```diff
@@ -45,4 +47,116 @@ describe("Standard Article with new ads hidden behind feature flags", () => {
+        displayNewAds
+        targetingData={targetingData(StandardArticle.id, "article")}
+      />
+    )
+
+    expect(canvas.props().adDimension).toEqual("970x250")
+    expect(canvas.props().adUnit).toEqual("Desktop_TopLeaderboard")
+    expect(canvas.props().displayNewAds).toBe(true)
+    expect(canvas.props().targetingData).toEqual({
+      is_testing: true,
+      page_type: "article",
+      post_id: "594a7e2254c37f00177c0ea9",
+    })
+    expect(canvas).toHaveLength(1)
+    canvas.find({ className: "htl-ad" })
+    canvas.find({ "data-sizes": "970x250" })
+    canvas.find({ "data-eager": true })
+    canvas.find({ "data-unit": "Desktop_TopLeaderboard" })
+  })
+
+  it("renders the panel component with the correct data and properties on standard articles", () => {
+    const panel = mount(
+      <NewDisplayPanel
+        adDimension={AdDimension.Desktop_RightRail1}
+        adUnit={AdUnit.Desktop_RightRail1}
+        displayNewAds
+        targetingData={targetingData(StandardArticle.id, "article")}
+      />
+    )
+
+    expect(panel.props().adDimension).toEqual("300x250")
+    expect(panel.props().adUnit).toEqual("Desktop_RightRail1")
+    expect(panel.props().displayNewAds).toBe(true)
+    expect(panel.props().targetingData).toEqual({
+      is_testing: true,
+      page_type: "article",
+      post_id: "594a7e2254c37f00177c0ea9",
+    })
+    expect(panel).toHaveLength(1)
+    panel.find({ className: "htl-ad" })
+    panel.find({ "data-sizes": "300x250" })
+    panel.find({ "data-eager": true })
+    panel.find({ "data-unit": "Desktop_RightRail1" })
+  })
+
+  it("renders the panel component with the correct data and properties on standard articles on mobile", () => {
+    const panel = mount(
+      <NewDisplayPanel
+        adDimension={AdDimension.Mobile_InContentMR1}
+        adUnit={AdUnit.Mobile_InContentMR1}
+        displayNewAds
+        targetingData={targetingData(StandardArticle.id, "article")}
+      />
+    )
+
+    expect(panel.props().adDimension).toEqual("300x250")
+    expect(panel.props().adUnit).toEqual("Mobile_InContentMR1")
+    expect(panel.props().displayNewAds).toBe(true)
+    expect(panel.props().targetingData).toEqual({
+      is_testing: true,
+      page_type: "article",
+      post_id: "594a7e2254c37f00177c0ea9",
+    })
+    expect(panel).toHaveLength(1)
+    panel.find({ className: "htl-ad" })
+    panel.find({ "data-sizes": "300x250" })
+    panel.find({ "data-eager": true })
+    panel.find({ "data-unit": "Mobile_InContentMR1" })
+  })
+
+  it("renders the canvas component with the correct data and properties on standard articles on mobile", () => {
+    const canvas = mount(
+      <NewDisplayCanvas
+        adDimension={AdDimension.Mobile_TopLeaderboard}
+        adUnit={AdUnit.Mobile_TopLeaderboard}
+        displayNewAds
+        targetingData={targetingData(StandardArticle.id, "article")}
+      />
+    )
+
+    expect(canvas.props().adDimension).toEqual("300x50")
```

Comment:
It looks like this spec is failing on CI. It received `"300x250"` instead of `"300x50"`.